### PR TITLE
Windurst Waters (S) Quest - Redeeming Rocks

### PR DESCRIPTION
**Pull request by hooksta4**
Orginal Credit goes to @Lusiphur. Converted from DSP to TPZ.

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by TeoTwawki** [comment]
That Mithra is clearly underage :police_officer:  as funny as that is should prolly edit that pr title

Some tabs snuck in your indents. you want 4 spaces, per the style guide.

**Comment by tankfest** [comment]
click bait

**Comment by hooksta4** [comment]
Removed the tabs I found... and changed the name.

**Comment by TeoTwawki** [comment]
> hooksta4 wrote: image

Me:
image

**Comment by hooksta4** [comment]
@TeoTwawki ❤️

File: scripts/zones/Vunkerl_Inlet_[S]/npcs/qm4.lua [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+require("scripts/globals/keyitems")
+require("scripts/globals/quests")
+-----------------------------------
+
+function onTrade(player, npc, trade)
+end
+
+function onTrigger(player, npc)
+    local RedeemingRocks = player:getQuestStatus(CRYSTAL_WAR, tpz.quest.id.crystalWar.REDEEMING_ROCKS)
+    local RocksProg = player:getCharVar("RedeemingRocksProg")
```
Comment (tankfest) [inline scripts/zones/Vunkerl_Inlet_[S]/npcs/qm4.lua]: You can delete these two variables as they are only called once each in the function.

`if player:getQuestStatus(CRYSTAL_WAR, tpz.quest.id.crystalWar.REDEEMING_ROCKS) == QUEST_ACCEPTED and player:getCharVar("RedeemingRocksProg") == 3 then`

Also as per the style guide you want to eliminate parenthesis from your if statements unless they are used for order of operations.

**Review by 59blargedy — APPROVED** [review]

**Pull request merged**